src/reporters/base-reporter.js

```diff
@@ -170,6 +175,27 @@ export default class BaseReporter {
     }
   }
 
+  getSizeUnit(iterations: number) {
+    switch(iterations) {
+      case 0:
+        return 'bytes';
+      case 1:
+        return 'KB';
+      case 2:
+        return 'MB';
+      case 3:
+        return 'GB';
+    }
+  }
+  convertBytes(bytes: number, iterations: number): string {
+    return bytes < 100
+    ? `${bytes.toFixed(2)} ${this.getSizeUnit(iterations)}`
+    : this.convertBytes(bytes / 1000, ++iterations);
+  }
+
+  getTotalNetworkUsage(): number {
+    return this.networkUsage > 0 ? this.convertBytes(this.networkUsage, 0) : 0;
```

Comment:
Do you really need the ternary? I think that module I referred about should be good enough :)

Reply:
Maybe not :) I'll give [humanize-bytes](https://yarnpkg.com/en/package/humanize-bytes) a look and see what I can do with it

Reply:
So, that lib you linked, humanize-bytes, only does "human -> bytes" and not the other way around, per their docs.

```javascript
var bytes = require('humanize-bytes');
bytes('1kb') // => 1024
bytes(1024) // => 1024
```

Poking around, not wanting to reinvent the wheel, I found [bytes](https://github.com/visionmedia/bytes.js), which seemed to have some ideal features that I was implementing, i.e. `decimalPlaces`